_includes/team.html

```diff
@@ -1,20 +1,20 @@
 <section id="team" class="team text-center bg-light">
  <div class="container mb-3">
-  <h1 class="mt-5 mb-3">Team</h1>
+  <h1 class="mt-5 mb-5">Team</h1>
   <div class="row">
    {% for leader in site.team.leaders %}
-    <div class="leader col-lg-4 col-sm-6 card">
+    <div class="leader team-col card">
```

Comment:
Is this correct?
Please see my comment on the PR

Reply:
@nitrosx "team-col" was a costum css class but I removed it and aligned the team div in center instead. It should now look like this:

![Capture](https://github.com/SciCatProject/scicatproject.github.io/assets/16343312/c7930d92-b02c-4fff-b8f4-90313f2dfb5f)